app/controllers/api/base_controller/manager.rb

```diff
@@ -122,14 +111,7 @@ def update_multiple_collections(is_subcollection, target, type, resources)
         processed = 0
         results = resources.each.collect do |r|
           next if r.blank?
-
-          rid = parse_id(r, type)
-          create_or_add_action = %w(create add).include?(action)
-          if rid && create_or_add_action
-            raise BadRequestError, "Resource id or href should not be specified for creating a new #{type}"
-          elsif !rid && !create_or_add_action
-            rid = parse_by_attr(r, type)
-          end
```

Comment:
This logic probably can't be removed, because in the routes you don't actually have all types of legacy requests routing to the new create method; that's because you need to handle passing an ID (on create... I know...), but doing so also ruins the point of the PR because you're trying to do something more conventional. 

This logic will have to remain until we're ready to actually state breaking changes and remove it.